code/modules/mob/living/simple_mob/subtypes/occult/living_statue.dm

```diff
@@ -143,7 +155,26 @@
 
 	return things
 
-
 /spell/aoe_turf/blindness/cast(list/targets, mob/user = usr)
 	for(var/mob/living/victim as anything in targets)
 		victim.Blind(4)
+
+/// Veil of Darkness Spell
+/spell/aoe_turf/veil_of_darkness
+	name = "Veil of Darkness"
+	desc = "You sheathe yourself in a powerful veil of darkness."
+
+	override_base = "grey"
+	hud_state = "wiz_smoke"
+
+	message = "<span class='notice'>You call upon the void.</span>"
+
+	charge_max = 1200
+	cooldown_min = 1200
+
+/spell/aoe_turf/veil_of_darkness/cast(list/targets, mob/user = usr)
+	playsound(usr.loc, 'sound/effects/bamf.ogg', 50, 1, 5)
+	var/obj/item/statue_darkness/S = new
+	usr.contents.Add(S)
+	S.set_light(5, -10, "#FFFFFF")
+	QDEL_IN(S, 2 SECONDS)
```

Comment:
usr is banned, why is it using that?
also you don't need the .loc in playsound, should just be the atom the sound's playing at.

Reply:
Fixed!